### ISSUE TITLE
Python update above version 2.7.9 needed for using TLS/SSL 1.2

### ISSUE THREAD

**Issue by sbaghdadi** [koying/SPMC#568]
Python lower than 2.7.9 doesn't supports TLS 1.2 but more an more webpages are upgrading to this version. Python above 2.7.9 is able to open pages with TLS 1.2. In kodi 17 this is already included. 

For example movie4k.to in xstream addon is not working anymore.

More information to this issue:

http://xstream-addon.square7.ch/showthread.php?tid=814

So please update python in spmc. Thank you in advance!

**Comment by humiboy** [comment]
@koying

please please add Python 2.7.9 in SPMC

**Comment by humiboy** [comment]
any news to work on support for Python 2.7.9?

**Comment by humiboy** [comment]
any news?

**Comment by sbaghdadi** [comment]
*bump*

**Comment by 99rook99** [comment]
If you look at the commits that koying has done since the middle of March, you will see that python has been updated to 2.7

**Comment by ndsports316** [comment]
I don't believe that is the case. I was reading on the forums a good test of phyton is to try to launch the movie4k.to addon and see if you get a tsl error. I get the same error on kodi 16.1 and Spmc 16.6 and do not get that error on kodi 17.1. I've also done multiple side by side scans and far more HD sources are populated on kodi 17.1. So I am not convinced that SPMC has been updated with the new python. 
![tls_1 2_f2](https://cloud.githubusercontent.com/assets/26742203/24636784/47468f4e-18a2-11e7-923e-1842cd0f7cbb.jpg)
![tls_1 2_g](https://cloud.githubusercontent.com/assets/26742203/24636786/48f1e94c-18a2-11e7-88b5-888b501447af.jpg)

**Comment by 99rook99** [comment]
Well, since version 16.6 was released 1 March and the first commit happened 11 March, you wouldn't have python 2.7. Now if you really wanted the version with 2.7, you could just compile the latest github version

**Comment by pickett2k** [comment]
So is this issue fixed and when will it be included with the latest app?

This version is way better than kodi 17 on my device, just need this https thing sorted out and no reason to use kodi anymore then

**Comment by ndsports316** [comment]
Koying released a patch already - see link here [topic 863](https://github.com/koying/SPMC/issues/863). It works I have been using it.
